### DISCUSSION TITLE
Rearranging Block Categories

### DISCUSSION THREAD

**Discussion by bobby-sills** [ScratchAddons/ScratchAddons#4971] [Ideas]
For me, it's kinda annoying that the variables and custom blocks are so far down in the categories when their probably the ones I use the most. I think it would be a useful feature to be able to rearrange the block categories so that the variables or custom blocks are at the top instead of motion. Please add this feature.

**Comment by DNin01** [comment]
This could probably become a new addon! If you would like to propose a new addon to help with this or suggest another specific possible solution, you can [create an issue](https://github.com/ScratchAddons/ScratchAddons/issues/new/choose).
